uPortal-webapp/src/main/resources/layout/theme/respondr/regions.xsl

```diff
@@ -292,19 +292,14 @@
     <!-- ========== TEMPLATE: FOOTER FIRST REGION =============== -->
     <!-- ======================================================== -->
     <!--
-     | This template renders region intended to hold the site navigation.
+     | This template renders the region intended to hold the sitemap.
```

Comment:
We should remove this comment;  some humans may have this intention, but the intention is not inherent in the theme.